client/analytics/settings/historical-data/actions.js

```diff
@@ -0,0 +1,82 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+
+function HistoricalDataActions( {
+	customersProgress,
+	customersTotal,
+	hasImportedData,
+	inProgress,
+	ordersProgress,
+	ordersTotal,
+} ) {
+	const getActions = () => {
+		// An import is currently in progress
+		if ( inProgress ) {
+			return (
+				<Fragment>
+					<Button
+						className="woocommerce-settings-historical-data__action-button"
+						isPrimary
+						onClick={ () => null }
+					>
+						{ __( 'Stop Import', 'woocommerce-admin' ) }
+					</Button>
+					<div className="woocommerce-setting__help woocommerce-settings-historical-data__action-help">
+						{ __(
+							'Imported data will not be lost if the import is stopped.',
+							'woocommerce-admin'
+						) }
+						<br />
+						{ __(
+							'Navigating away from this page will not affect the import.',
+							'woocommerce-admin'
+						) }
+					</div>
+				</Fragment>
+			);
+		}
+
+		// Has no imported data
+		if ( ! hasImportedData ) {
+			return (
+				<Button isPrimary onClick={ () => null }>
+					{ __( 'Start', 'woocommerce-admin' ) }
+				</Button>
+			);
+		}
+
+		// Has imported all possible data
+		if ( customersProgress === customersTotal && ordersProgress === ordersTotal ) {
+			return (
+				<Button isDefault onClick={ () => null }>
+					{ __( 'Import Additional Historical Data', 'woocommerce-admin' ) }
```

Comment:
This text sounds a bit odd to me, like there's more that can be imported.  I kind of expected something like `Re-import Data` since everything has already been imported here. /cc @LevinMedia 

Also, do we want to show the delete option here?

Reply:
> This text sounds a bit odd to me, like there's more that can be imported.

Agree with that, it was also confusing for me. I changed it for now. We can undo it or change it again in case @LevinMedia prefers another copy.

> Also, do we want to show the delete option here?

Actually in #1851 there is a _Delete_ button that was not in Figma designs. I think it makes sense adding it.

Both changes are included in b2f47e436bd446934248c8807bdd4730c4945f5b.